simphony/io/tests/test_h5_mesh.py

```diff
@@ -332,6 +350,15 @@ def test_get_all_edges_iterator(self):
 
         self.assertItemsEqual(iedges_id, euids)
 
+        mesh_points = [p.uid for p in self.mesh.iter_points()]
```

Comment:
We should check that the specific uuids in each edge.points are equal to the uuids there where given at initialization. 
something like
```python
for index, edge in self.mesh.iter_edges(euids):
    self.assertItemEqual(edge.points, edges[index].points)
```